pkg/ui/src/redux/alerts.spec.ts

```diff
@@ -350,6 +355,26 @@ describe("alerts", function() {
         });
       });
     });
+
+    describe("email signup for release notes alert", () => {
+      it("initialized with default 'false' (hidden) state", () => {
+        const settingState = emailSubscriptionAlertLocalSetting.selector(state());
+        assert.isFalse(settingState);
+      });
+
+      it("dismissed by by alert#dismiss", async () => {
+        // set alert to open state
+        dispatch(emailSubscriptionAlertLocalSetting.set(true));
+        const openState = emailSubscriptionAlertLocalSetting.selector(state());
+        assert.isTrue(openState);
+
+        // dismiss alert
+        const alert = emailSubscriptionAlertSelector(state());
+        await alert.dismiss(dispatch, state);
+        const dismissedState = emailSubscriptionAlertLocalSetting.selector(state());
```

Comment:
Do you think this should also be `openState`? It reads a bit weird to dismiss the alert and then check that `dismissedState` `isFalse`. What do you think?

Reply:
Hah, good point! Thank you!